prow/config.yaml

```diff
@@ -52,19 +65,18 @@ presubmits:
     rerun_command: "/test pull-kops-e2e-kubernetes-aws"
     trigger: "(?m)^/test( all| pull-kops-e2e-kubernetes-aws),?(\\s+|$)"
   kubernetes/kubernetes:
-  - name: pull-kubernetes-bazel-build
-    agent: kubernetes
-    context: pull-kubernetes-bazel-build
-    always_run: true
-    rerun_command: "/test pull-kubernetes-bazel-build"
-    trigger: "(?m)^/test( all| pull-kubernetes-bazel-build),?(\\s+|$)"
-    skip_branches:
-    - release-1.4
-    - release-1.5
-    - release-1.6  # doesn't have BUILD files under //vendor/k8s.io
+  - <<: &pull-kubernetes-bazel-build-spec
```

Comment:
this might be confusing with the `spec` field. not sure what a less confusing name would be though

Reply:
changed to `-job`.